From 263a064acc3e48ddda9886061177f535aaf7f221 Mon Sep 17 00:00:00 2001
From: =?utf-8?q?R=C3=A9mi=20Denis-Courmont?= <rdenis@simphalempin.com>
Date: Thu, 21 Aug 2008 18:23:55 +0300
Subject: [PATCH] Use VLC_ENOMEM instead of VLC_EGENERIC.

(cherry picked from commit c6a78e1b9e0cd1b87783b024f66ee440965053d5)

Not cherry-picked:

	modules/demux/rawvid.c
---
 modules/demux/tta.c |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/modules/demux/tta.c b/modules/demux/tta.c
index a277fff..5d8ff50 100644
--- a/modules/demux/tta.c
+++ b/modules/demux/tta.c
@@ -107,7 +107,7 @@ static int Open( vlc_object_t * p_this )
     p_demux->p_sys = p_sys = malloc( sizeof( demux_sys_t ) );
     
     if( !p_sys )
-        return VLC_EGENERIC;
+        return VLC_ENOMEM;
 
     /* Read the metadata */
     es_format_Init( &fmt, AUDIO_ES, VLC_FOURCC( 'T', 'T', 'A', '1' ) );
@@ -132,7 +132,7 @@ static int Open( vlc_object_t * p_this )
     if( !p_seektable )
     {
         free( p_sys );
-        return VLC_EGENERIC;
+        return VLC_ENOMEM;
     }
 
     stream_Read( p_demux->s, p_seektable, i_seektable_size );
@@ -141,7 +141,7 @@ static int Open( vlc_object_t * p_this )
     {
         free( p_seektable );
         free( p_sys );
-        return VLC_EGENERIC;
+        return VLC_ENOMEM;
     }
 
     for( i = 0; i < p_sys->i_totalframes; i++ )
@@ -157,7 +157,7 @@ static int Open( vlc_object_t * p_this )
         free( p_sys->pi_seektable );
         free( p_seektable );
         free( p_sys );
-        return VLC_EGENERIC;
+        return VLC_ENOMEM;
     }
     memcpy( (uint8_t*)fmt.p_extra, p_header, 22 );
     memcpy( (uint8_t*)fmt.p_extra+22, p_seektable, fmt.i_extra -22 );
-- 
1.5.6.5

